docs/source/getting_started/platform.rst

```diff
@@ -194,13 +194,47 @@ need to look like this:
         ]
     }
 
+If you don't have your data in coco format yet, but e.g. as a pandas dataframe,
+you can use a simple script to translate it to the coco format:
+
+.. code-block:: python
+
+    import numpy as np
+    import pandas as pd
+
+    from lightly.utils import save_custom_metadata
+
+    # Let's assume you have your data in form of a pandas dataframe
+    df = pd.DataFrame({
+        "image0.jpg": [3, "cloudy", 20.3],
+        "image1.jpg": [1, "rainy", 15.0]
+    })
```

Comment:
In most cases you have a pandas dataframe with column names (for example when you load it from a csv). I would include column names here as well and then use them to set the metadata names as well.

Reply:
Good point, I added that.